go/dhcp/main.go

```diff
@@ -586,6 +652,21 @@ func (h *Interface) ServeDHCP(ctx context.Context, p dhcp.Packet, msgType dhcp.M
 						}
 					}
 
+					// Add options on the fly from pffilter
+					for key, value := range info.(map[string]interface{}) {
```

Comment:
Same comment about code duplication for this

https://github.com/inverse-inc/packetfence/pull/3784/files#diff-2a21322cf6c4918fdce4d3b8c0736ef4R492

Goal is to make this as lean as @lzammit